sourcegit/watcher.py

```diff
@@ -0,0 +1,73 @@
+"""
+Watch CI and report results back to upstream.
+"""
+import logging
+
+import github
+
+from onegittorulethemall.services.pagure import PagureService
+
+
+logger = logging.getLogger(__name__)
+# FIXME: create a global config and move this data in there
+package_mapping = {
+    "python-docker": {
+        "source-git": "TomasTomecek/docker-py-source-git"
+    }
+}
+
+
+class Holyrood:
+    """ such a good gin """
```

Comment:
This class needs some cleaning and name-polishing (`Holyrood`, `self.g`, `ps`). If I am right, this is at least partially done in the second PR, isn't it?

Reply:
yup, I changed it in [2]